randomizer.py

```diff
@@ -1835,7 +1893,32 @@ def RandomizeDominion(setNames=None, options=None):
     if includeHorse:
         additionalCards.add("Menagerie: Horse")
     if includeLoot:
-        additionalCards.add("(Plunder: Loot Deck)")
+        landscapeList.append("(Plunder: Loot Deck)")
+
+    # Assign Traits to selected cards
+    selectedTraits = Traits.intersection(landscapeList)
+    if selectedTraits:
+        eligibleCards = resultSet - NoTraitCards
+        for trait in selectedTraits:
+            landscapeList.remove(trait)
+            if not eligibleCards:
+                break
+            traitCard = random.sample(eligibleCards, 1)[0]
+            eligibleCards.remove(traitCard)
+            landscapeList.append("{} (on {})".format(trait, traitCard))
+
+    # Obelisk support
+    obeliskPicked = set(landscapeList) & Empires.cards("Obelisk")
```

Comment:
Rather than converting `landscapeList` into a set, this can be flipped:

```python
obeliskPicked = Empires.cards("Obelisk").intersection(landscapeList)
```